src/pages/CommunicatingWithGraphQLAPI.js

```diff
@@ -10,36 +10,39 @@ const content = markdown(markdownOptions)`
 
 In this section, we’ll explain the core concepts you need to know when working with a GraphQL API. 
 
+<!--
```

Comment:
Same as in RNX - this is a syntax error, let's either update it or remove it

Reply:
Done 🙌